src/validations.ts

```diff
@@ -12,8 +13,12 @@ export function validateEmail(
 
   if (!emailRegex.test(value)) {
     return commonErrorKey;
+  } else {
+    const matchAccents = value.match(accentRegex);
+    if (matchAccents !== null && matchAccents[0] !== value) {
```

Comment:
Why not use test instead of match? 
```suggestion
    if (accentRegex.test(value)) {
```

Reply:
https://www.w3schools.com/jsref/jsref_regexp_test.asp

Reply:
I've talked with @fcoronelmakingsense and the aim of this regex it to detect a valid format, when it fails in detecting the whole email as correct that's when we have an accent. I demand tests for this 😄

Reply:
Because with test don't work. always has true value. With match return array with the all matches